filters/quantitative_ques/filter.py

```diff
@@ -0,0 +1,24 @@
+import spacy
+import sys
+sys.path.append('/home/tanay/Documents/GEM/NL-Augmenter')
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+
+class QuantitativeQuestion(SentenceOperation):
+    tasks = [TaskType.QUESTION_ANSWERING, TaskType.QUESTION_GENERATION]
+    languages = ["en"]
+
+    def __init__(self):
+        super().__init__()
+        self.nlp = spacy.load("en_core_web_sm")
+
+    def filter(self, sentence: str = None) -> bool:
+        tokenized = self.nlp(sentence, disable=["parser", "tagger", "ner"])
+        if 'how' == tokenized[0].text.lower() and tokenized[1].text.lower() in ['many','far','long','much','old']:
+            return True
+        else:
+            return False
```

Comment:
You could maybe make this word list a class variable (e.g `self.quant_words`) instead of directly writing it out in the loop body here. This way, you can easily keep updating the list without touching your filter() function body.

This is just a loosely phrased thought, and no need to work on this, but it would be super-cool if this transformation class can also accept some way of expanding these basic quant words, maybe even not entirely specified in advance. E.g users can pass paraphrase tables into the class or to an expand() function, which augments this set.

Reply:
You could also make some easy additions to this list to make it have better coverage - e.g broad, deep, tall, wide, heavy etc...

One class of quantitative questions not covered by your perturbation is maximum/minimum/comparison queries - might be worthwhile mentioning this somewhere e.g in limitations...